nntrainer/tensor/char_tensor.cpp

```diff
@@ -141,6 +162,25 @@ void *CharTensor::getData(size_t idx) const {
   return data->getAddr<int8_t>() + offset + idx;
 }
 
+void *CharTensor::getScale() const {
+  if (!data)
+    return nullptr;
+
+  data->validate();
+  return ((int8_t *)getData()) + size();
+}
+
+void *CharTensor::getScale(size_t idx) const {
+  NNTR_THROW_IF(idx > scale_size(), std::invalid_argument)
+    << "Tensor::getScale() index is not valid";
+
+  if (!data)
+    return nullptr;
+
+  data->validate();
+  return ((int8_t *)getScale()) + idx;
```

Comment:
Isn't it  `((float *)getScale()) + idx`?

Reply:
yes, you're right! thank you 👍